paddle/scripts/docker/README.md

```diff
@@ -166,3 +166,17 @@ docker tag myapp me/myapp
 docker push
 kubectl ...
 ```
+
+### Reading source code with woboq codebrowser
+If you are interesting of PaddlePaddle source code, [Woboq codebrowser](https://github.com/woboq/woboq_codebrowser) is a suitable tool.
```

Comment:
For developers who are interested in the C++ source code, please use `-e "WOBOQ=ON"` to enable building C++ source code into HTML pages using [Woboq codebrowser](https://github.com/woboq/woboq_codebrowser)